### PR TITLE
Fixes CI

### LINKED ISSUE #173: Get CI Working again

### PR DESCRIPTION
**Pull request by ashfurrow**
Fixes #173.

### CODE REVIEW

**Comment by ashfurrow** [comment]
Looks like something was missed in https://github.com/ashfurrow/Moya/pull/172

**Comment by colinta** [comment]
There's no way to remove Demo/Pods from the repo, is there?  Sure is a lotta noise.

**Comment by ashfurrow** [comment]
Ha, yeah @colinta I hear you. From what I understand of Carthage, which is by choice not a lot, it's necessary.

**Comment by colinta** [comment]
Hahaha, well I'm with you on that.  Seems like a classic case of fixing what ain't broke, if you ask me.

**Comment by colinta** [comment]
NIIIICE

File: Demo/DemoTests/MoyaProviderSpec.swift [hunk 1]
```diff
@@ -277,7 +277,7 @@ class MoyaProviderSpec: QuickSpec {
                         let parallelCount = 10
                         let observables = Array(0..<parallelCount).map { _ in provider.request(target) }
                         var completions = Array(0..<parallelCount).map { _ in false }
-                        let queue = dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0)
+                        let queue = dispatch_queue_create("testing", DISPATCH_QUEUE_CONCURRENT)
```
Comment (ashfurrow) [inline Demo/DemoTests/MoyaProviderSpec.swift]: @alcarvalho I think this seems more accurate to the intention of the test, but please do let me know.

Comment (alcarvalho) [inline Demo/DemoTests/MoyaProviderSpec.swift]: @ashfurrow yeah, it is... I just avoided creating a new queue due to my lack of knowledge on memory management of queues with Swift. I didn't know if I would need to release it and all that. But that's nice!

Comment (ashfurrow) [inline Demo/DemoTests/MoyaProviderSpec.swift]: Nah, no problem! I had to look it up, too. `dispatch_release` isn't even exposed in Swift.

**Pull request merged**